src/libs/interfaces/generator/parser.cpp

```diff
@@ -588,6 +591,14 @@ InterfaceParser::parse()
 				throw InterfaceGeneratorInvalidContentException("no name for message");
 			}
 			msg_name = attr->get_value();
+#ifdef CHECK_MESSAGE_TYPE_SIZE
+			if (msg_name.length() > INTERFACE_MESSAGE_TYPE_SIZE_ - 1 - 7) {
```

Comment:
It's still hard to guess what the hard-coded 7 is supposed to mean. More explicit check:
```suggestion
			if (msg_name.length() + std::string("Message").length() > INTERFACE_MESSAGE_TYPE_SIZE_ - 1) {
```